ui/mirage/factories/allocation.js

```diff
@@ -13,7 +13,7 @@ const REF_TIME = new Date();
 export default Factory.extend({
   id: (i) => (i >= 100 ? `${UUIDS[i % 100]}-${i}` : UUIDS[i]),
 
-  jobVersion: 1,
+  jobVersion: 0,
```

Comment:
TBD: jobVersion is set to 1 in our tests, but Nomad jobVersions start at 0. Setting this to 0 to try to make Mirage a little closer to real life. If this breaks a lot of things, will revert.